pkg/storage/access_token_file.go

```diff
@@ -0,0 +1,25 @@
+package storage
+
+import "github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+
+type AccessTokenFile struct {
+	TokenFilePath	string
+	Data	[]byte
+}
+
+func NewAccessTokenFile(config config.Config) (token *AccessTokenFile, err error) {
+	return
+}
+
+func (token *AccessTokenFile) Read() (Data []byte, err error) {
```

Comment:
If we are not changing the struct it should pass by value not by reference (without *)

Reply:
also here pass by value